brvs-rules/src/test/java/iroha/validation/rules/CheckWhitelistRule.java

```diff
@@ -0,0 +1,110 @@
+package iroha.validation.rules;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import iroha.protocol.Commands;
+import iroha.protocol.QryResponses.QueryResponse;
+import iroha.protocol.Queries.Query;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import java.security.KeyPair;
+import java.util.Collections;
+import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import org.junit.jupiter.api.Test;
+
+class CheckWhitelistRule {
```

Comment:
By convention this test must be called `CheckWhitelistRuleTest` and moved to the `iroha.validation.rules.impl.whitelist` package.